src/config/routes.rb

```diff
@@ -3,7 +3,7 @@
 
   get '/home' => 'pages#home', as: :home_page
 
-  resources :sessions, :only => [:index, :show, :new, :create, :update, :edit] do
+  resources :sessions, :only => [:index, :show, :new, :create, :update, :edit, :destroy] do
```

Comment:
Now that there's a `:destroy` action, you could remove all of the `:only` options:

`resources :sessions`